devices/danfoss.js

```diff
@@ -45,7 +45,8 @@ module.exports = [
                 .withSystemMode(['heat']).withRunningState(['idle', 'heat'], ea.STATE),
             exposes.numeric('external_measured_room_sensor', ea.ALL)
                 .withDescription('Set at maximum 3 hours interval but not more often than every 30 minutes and 0.1 ' +
-                    'degrees difference. Resets every 3hours to standard. e.g. 21C = 2100 (-8000=undefined).'),
+                    'degrees difference. Resets every 3hours to standard. e.g. 21C = 2100 (-8000=undefined).')
+                .withValueMin(-8000).withValueMax(35),
```

Comment:
Max should be `3500` I think? Given that 21C = 2100 according to the description.

Reply:
Same for other thermostats based on this design

Reply:
Should be -8000 to 3500.

Same for Hive and Popp